base/notify-celery-main-primary/celery-deployment.yaml

```diff
@@ -2,16 +2,16 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: celery-fargate
```

Comment:
this is odd - why did you use the celery-deploment-fargate file instead of the celery-deployment one?

Reply:
or did you and then delete fargate and this is Git madness?

Reply:
That is odd - I didn't... at least I didn't mean to

Reply:
Yeah so this resolves to: 
``` yaml   
app: celery-primary
    # profile: fargate
  name:  celery-primary
  namespace: notification-canada-ca
  ```

I don't know what the heck git thinks is happening